test/reports/bootstrap-voting/rest-gateway-0-rest.json

```diff
@@ -91,4 +91,4 @@
     "deploymentToolVersion": "abc",
     "lastUpdatedDate": "2021-05-23"
   }
-}
+}
```

Comment:
newline?

Reply:
Thanks, added the new lines back after lost in a geneation